py/janitor/bzr_store.py

```diff
@@ -351,34 +351,31 @@ async def create_web_app(
 async def main_async(argv=None):
     import argparse
 
-    parser = argparse.ArgumentParser(prog="janitor.bzr_store")
+    parser = argparse.ArgumentParser(prog="janitor.bzr_store", formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+    parser.add_argument("--port", type=int, help="Listen port", default=9929)
     parser.add_argument(
-        "--prometheus", type=str, help="Prometheus push gateway to export to."
+        "--public-port", type=int, help="Public listen port for a reverse proxy", default=9930
```

Comment:
This is intentionally a string so you can specify a service name here (e.g. "http")

Reply:
I didn't change `public-port` - its always been an int:

```console
$ git diff HEAD HEAD~ py/janitor/bzr_store.py | grep public-port
-        "--public-port", type=int, help="Public listen port for a reverse proxy", default=9930
+        "--public-port", type=int, help="Public listen port", default=9930
```

---

This is the same with `prometheus`, its always been an str:

```console
$ git diff HEAD HEAD~ py/janitor/bzr_store.py | grep prometheus
+        "--prometheus", type=str, help="Prometheus push gateway to export to."
-        "--prometheus", type=str, help="Prometheus push gateway to export to"
$
```